playlists-prod/word.yaml

```diff
@@ -111,6 +111,15 @@
   group: Content Controls
   api_set:
     WordApi: '1.7'
+- id: word-content-controls-insert-and-change-tracking-state
+  name: Get content controls by their change tracking state
+  fileName: insert-and-change-tracking-state.yaml
+  description: Inserts content controls then retrieves them by their change tracking state.
+  rawUrl: >-
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/10-content-controls/insert-and-change-tracking-state.yaml
+  group: Content Controls
+  api_set:
+    WordApi: '1.5'
 - id: word-images-insert-and-get-pictures
   name: Use inline pictures
```

Comment:
wrong text about images?

Reply:
@jipyua I'll confirm if that snippet has the correct name

Reply:
Confirmed that the name is correct - the snippet inserts an inline image then retrieves it